core/test_network/src/lib.rs

```diff
@@ -404,6 +404,9 @@ where
     shutdown: Option<JoinHandle<()>>,
     /// Iroha itself
     pub iroha: Option<Iroha<G, S, B>>,
+    /// Temp dir
+    // Note: last field to be dropped after Iroha (struct fields drops in FIFO RFC 1857)
```

Comment:
Oh, here you use the approach with fields drop order

Reply:
Yes, because we can't use `ManuallyDrop/Option/...` to drop field last one without wrapping other fields too.